### ISSUE TITLE
Update to support 16.04 

### ISSUE THREAD

**Issue by stefanreimers** [tim-sueberkrueb/ubuntu-hangups#70]
Hi,
are there any plans to update the Hangups app in OpenStore to support the upcoming 16.04 release? Reading the docs (https://forums.ubports.com/topic/1215/updating-apps-to-16-04) tells me there are some settings to change but I cannot tell how much work the rebuilding and submitting to the store is.

I would appreciate that very much as I am a frequent user of the app.

Thanks for the help.

**Comment by timsueberkrueb** [comment]
Hey @stefanreimers, thanks for the information.

Ubuntu Hangups is currently unmaintained because I'm unfortunately lacking the time and - as I'm personally not actively using Google Hangouts for chat messaging anymore - motivation to continue maintaining it. So there won't be any new features or even bug fixes from my side - at least for now.

That being said, if anyone wants to pick it up, I'm more than happy to help getting started. I will look into repackaging it for 16.04 and would also review and accept contributions for this matter.

**Comment by timsueberkrueb** [comment]
Some AppArmor issues I ran into:
```
getrlimit(RLIMIT_NOFILE) failed
propsReply "An AppArmor policy prevents this sender from sending this message to this recipient; type=\"method_call\", sender=\":1.238\" (uid=32011 pid=13712 comm=\"/usr/lib/qt5/bin/qmlscene $@ Main.qml \") interface=\"org.freedesktop.DBus.Properties\" member=\"GetAll\" error name=\"(unset)\" requested_reply=\"0\" destination=\"org.freedesktop.NetworkManager\" (uid=0 pid=1704 comm=\"NetworkManager \")"
nmReply "An AppArmor policy prevents this sender from sending this message to this recipient; type=\"method_call\", sender=\":1.238\" (uid=32011 pid=13712 comm=\"/usr/lib/qt5/bin/qmlscene $@ Main.qml \") interface=\"org.freedesktop.NetworkManager\" member=\"GetDevices\" error name=\"(unset)\" requested_reply=\"0\" destination=\"org.freedesktop.NetworkManager\" (uid=0 pid=1704 comm=\"NetworkManager \")"
"Object path cannot be empty"
```

**Comment by zubozrout** [comment]
@timsueberkrueb Yeah, I've got the same errors in console when running my app but it doesn't seem to be affecting any functionality fortunately. https://paste.ubuntu.com/p/DSs729tqCr/

**Comment by timsueberkrueb** [comment]
Okay, interesting. The app crashed, so I was expecting this deny was related, but then it might actually be a different reason.
